tracer/src/Datadog.Trace/Tagging/OpenTelemetryTags.cs

```diff
@@ -0,0 +1,48 @@
+// <copyright file="OpenTelemetryTags.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.SourceGenerators;
+
+namespace Datadog.Trace.Tagging
+{
+    internal partial class OpenTelemetryTags : CommonTags
```

Comment:
Should I copy over the `AnalyticsSampleRate` and `SetAnalyticsSampleRate` from `InstrumentationTags`?

https://github.com/DataDog/dd-trace-dotnet/blob/b07c2590ed47eb547ceb1e787a721fa6ef194239/tracer/src/Datadog.Trace/Tagging/InstrumentationTags.cs#L17-L27

I didn't inherit from `InstrumentationTags` as the `SpanKind` needs to be settable

Reply:
Hmm I'm not sure what we're supposed to do with `SetAnalyticsSampleRate` and honestly it's been a while since I really understood what this is doing. 🆘 @lucaspimentel 

Based on my current understanding, we would set this based on the SpanKind:
- Client/Producer: `tags.SetAnalyticsSampleRate(IntegrationId, tracer.Settings, enabledWithGlobalSetting: false)`
- Server/Consumer: `tags.SetAnalyticsSampleRate(IntegrationId, tracer.Settings, enabledWithGlobalSetting: true)`
- Internal: ???

Reply:
Internal spans would also be `enabledWithGlobalSetting: false`. Only "server" spans should have Analytics enabled with the global setting, and I think we can consider "consumer" spans as similar type of span (they're both the "entry point" into a service from an upstream service).